lib/artifactory/resources/repository.rb

```diff
@@ -81,6 +81,7 @@ def find(name, options = {})
     attribute :rclass, 'local'
     attribute :snapshot_version_behavior, 'non-unique'
     attribute :suppress_pom_consistency_checks, false
+    attribute :url, ''
```

Comment:
Do we want the default value to be a blank string or would `nil` be more appropriate? Not providing a default value is the same as an explicit `nil`:
https://github.com/martinmosegaard/artifactory-client/blob/c661f3c2c87b38fd3167495567b80afcbba8d2c6/lib/artifactory/resources/base.rb#L45